svg/MIL_STD_2525D_Symbols/Legacy/README.md

```diff
@@ -0,0 +1,29 @@
+### Creating Full-Frame Legacy Icons using 2525D format.
+
+*This process was done using Adobe Illustrator CS6.*
+
+1. Open these 4 [template SVG's](https://github.com/Esri/joint-military-symbology-xml/tree/master/svg/MIL_STD_2525D_Symbols/Frames/Template) in separate views in Illustrator. 
```

Comment:
Should be updated to address the other, more common template cases: BoundingOctagon_H BoundingOctagon_V BoundingOctagon